Applications/Spire/Source/Ui/ListItem.cpp

```diff
@@ -9,7 +9,8 @@ using namespace Spire;
 using namespace Spire::Styles;
 
 ListItem::ListItem(QWidget* component, QWidget* parent)
-    : QWidget(parent) {
+    : QWidget(parent),
+      m_is_selected(false) {
   setFocusPolicy(Qt::StrongFocus);
   auto layout = new QHBoxLayout(this);
   layout->setContentsMargins({});
```

Comment:
The specs were updated to place a ```Box``` inside of the ```Button``` but this implementation isn't doing that.